openprescribing/measure_definitions/edoxaban.json

```diff
@@ -0,0 +1,69 @@
+    "In January 2022 the NHS in England <a href='https://www.england.nhs.uk/wp-content/uploads/2022/01/B1279-national-procurement-for-DOACs-commissioning-recommendations-v1.pdf'>",
+    "announced a national procurement programme for DOACs, which states:</a></p>",
+    "<ul>",
+    "<li>For patients commencing treatment for AF: subject to the criteria specified in the relevant NICE technology appraisal guidance, clinicians should use ",
+    "edoxaban where this is clinically appropriate. If edoxaban is contraindicated or not clinically appropriate for the specific patient then, subject to the criteria ",
+    "specified in the relevant NICE technology appraisal guidance, clinicians should then consider rivaroxaban first, then apixaban or dabigatran.",
+    "<li>For patients already prescribed a DOAC for the treatment of AF: subject to the criteria specified in the relevant NICE technology appraisal guidance, ",
+    "commissioners may wish to consider developing local policy to review patients currently prescribed apixaban, rivaroxaban or dabigatran, where clinically appropriate.</li>",
+    "</ul>",
+    "</p>",
+    "The 2022-2023 PCN Investment and Impact Fund (IIF) <a href='https://www.england.nhs.uk/wp-content/uploads/2022/03/B1357-Network-Contract-Directed-Enhanced-Service-contract-specification-2022-23-primary-care-network-requireme.pdf'>",
+    "now includes the following indicator: </a>",
+    "<ul>",
+    "<li>CVD-06: Number of patients who are currently prescribed edoxaban, as a percentage of patients on the QOF Atrial Fibrillation register and with a CHA<sub>2</sub>DS<sub>2</sub>-VASc score of ",
+    "2 or more (1 or more for patients that are not female) and who are currently prescribed a direct-acting oral anticoagulant (DOAC).",
+    "</ul>",
+    "</p>",
+    "<b>Please note:</b> as prescribing data does not contain patient information or indication, this measure uses prescription items of DOACs prescribed for all indications in ",
+    "primary care, and therefore will not exactly match the IIF indicator.  OpenPrescribing.net measures also have a consistent way of visualising measures, where 'lower is better' ",
+    "and therefore in this case we show the proportion of DOACs which are NOT prescribed as edoxaban." 
+  ],
+  "tags": [
+    "cardiovascular",
+    "cost"
+  ],
+  "url": null,
+  "is_percentage": true,
+  "is_cost_based": false,
+  "low_is_good": true,
+  "numerator_type": "bnf_items",
+  "numerator_bnf_codes_filter": [
+    "0208020Y0 # Rivaroxaban",
+    "0208020Z0 # Apixaban",
+    "0208020X0 # Dabigatran etexilate"
+  ],
+ "denominator_type": "bnf_items",
+  "denominator_bnf_codes_filter": [
+    "0208020Y0 # Rivaroxaban",
+    "0208020AA # Edoxaban",
+    "0208020Z0 # Apixaban",
+    "0208020X0 # Dabigatran etexilate"
+  ],  
+"authored_by": [
+    "richard.croker@phc.ox.ac.uk"
+  ],
+  "checked_by": [
+    ""
+  ],
+  "date_reviewed": [
+    "2022-04-11"
+  ],
+  "next_review": [
+    "2023-03-14"
```

Comment:
do we have a method/scipt for systematically identifying when they are ready for review?

Reply:
Unfortunately not at the moment.  When I did a review of the measure definitions a few years ago I put the dates in in order to have exactly that, but obviously we haven't had the resource to fully implement.  Something to reconsider now.